packages/solarwinds-apm/CONFIGURATION.md

```diff
@@ -81,6 +81,42 @@ It is possible to export logs to the collector by explicitly enabling the featur
 - `pino`
 - `winston`
 
+### Transaction Names
+
+By default, the library will name transactions using a set of internal rules. However there are cases where the default naming logic doesn't fit. The `transactionName` option allows customising the behaviour.
+
+When set to a string literal, every transaction will use the name without exception. When set to a function, it will be called with the span object as its only parameter. The return value is used as transaction name, unless it is `undefined`, in which case the default naming logic is used.
```

Comment:
How about also noting that function and scheme approaches only work with the "local root span" (seems to be upstream term of choice for what we call entry span).